docs/source/commands.rst

```diff
@@ -246,3 +246,12 @@ it's name or id could be used:
   gohan client subnet create --network "Network Name"
   gohan client subnet create --network network-id
   gohan client subnet create --network_id network-id
+
+Custom commands
+---------------
+
+Any `custom actions <./schema.rst/#custom-actions-schema>`_ specified in schema are also supported as commands by gohan client and should be invoked as follows:
```

Comment:
I don't really know anything about creating docs from .rst format, so someone should please verify, that this link doesn't break